CHANGELOG.md

```diff
@@ -2,6 +2,69 @@ The Twilio Programmable Video SDKs use [Semantic Versioning](http://www.semver.o
 
 **Version 1.x will End of Life on September 8th, 2021.** Check [this guide](https://www.twilio.com/docs/video/migrating-1x-2x) to plan your migration to the latest 2.x version. Support for the 1.x version ended on December 4th, 2020.
 
+2.17.0 (In Progress)
+====================
+**Preflight API Pilot**
+-----------------------
+
+- This release adds a new experimental API `runPreflight` to help test the connectivity with twilio servers.
+The API connects two Peer connections using Twilio's turn servers. It publishes synthetic audio and video
+tracks from one participant and ensures that other participant receives media on those tracks.
+After successfully verifying the connectivity, it generates a report about statistics of the connection.
+
+**Few things to note:**
+- This function uses <a href="https://developer.mozilla.org/en-US/docs/Web/API/Web_Audio_API"> web  audio API's.</a>
+Browser's  autoplay policies sometimes require user action before accessing these APIs. Please ensure that this API
+is called in response to user action like a button click.
+- `preflightTest` emits `failed` event to indicate test failures. Please refer to
+[this guide](https://www.twilio.com/docs/video/build-js-video-application-recommendations-and-best-practices#connection-errors)
+for interpreting common errors.
+
+
+Report generated by `preflightTest` on `completed` can be used to determine the quality of connection. Some of the useful stats and their optimal values are listed below.
+
+| Stat                     |  Explanation | Optimal values
+| -------------------------|--------------|------------------|
+| jitter                   | Packets delay variation on audio tracks | < 30ms
+| packetLoss               | Packet loss as a percent of total packets sent | < 1%
+| rtt                      | Round trip time in milliseconds. | < 200ms
+
+The API also calculates an estimated quality score using jitter, packet loss and rtt values observed.
+The quality score can be one of:
+
+| Value                    | Meaning
+| -------------------------|--------------------------------|
+| 0                        | Failed
+| 1                        | Degraded
+| 2                        | Fair
+| 3                        | Good
+| 4                        | Great
+| 5                        | Excellent
```

Comment:
Not sure if I missed it, but is the report documented in our API docs?

Reply:
yes. here is a link to the doc: https://46634-46595751-gh.circle-artifacts.com/0/dist/docs/global.html#PreflightTestReport